test/integration/default/inspec/default_spec.rb

```diff
@@ -2,3 +2,25 @@
 
 # The InSpec reference, with examples and extensive documentation, can be
 # found at https://docs.chef.io/inspec/resources/
+
+describe user 'seagl' do
+  it { should exist }
+  its('home') { should cmp '/home/seagl' }
+end
+
+describe file '/etc/sudoers.d/seagl' do
+  it { should be_file }
+  its('mode') { should cmp 0440 }
+  its('owner') { should cmp 'root' }
+  its('group') { should cmp 'root' }
+  its(:content) { should match(/seagl ALL=\(root\) NOPASSWD:ALL/) }
+end
```

Comment:
A better integration test would be to run `sudo -l` as the `seagl` user and make sure it prints the output we expect.

Reply:
Just pushed a different test for this. How's it look now?